internal/schema/schema.go

```diff
@@ -545,15 +540,6 @@ func fetchIndexes(ctx context.Context, q *queries.Queries) ([]Index, error) {
 	return indexes, nil
 }
 
-func getIndexConstraintType(constraintType string) (IndexConstraintType, error) {
```

Comment:
Don't bother converting the constraint type string into an enum. The old method would cause schema extraction to fail if a new constraint type was added -- even if it wasn't used anywhere in the diffing code.